modApiServer/src/org/aion/api/server/rpc3/types/RPCTypesConverter.java

```diff
@@ -50,9 +62,9 @@ else if(decPattern.matcher(s.toString()).find()){
             }
         }
 
-        public static String encode(Long s){
+        public static Long encode(Long s){
```

Comment:
I don't understand why the encode method do Long object to Long object.

Reply:
I am using the Long type here instead of string so that the json library observes the correct typing for integer/long types. Otherwise they will be formatted as Strings.